libmamba/include/mamba/api/install.hpp

```diff
@@ -105,8 +105,12 @@ namespace mamba
 
         bool eval_selector(const std::string& selector, const std::string& platform);
 
-        yaml_file_contents
-        read_yaml_file(const Context& ctx, const std::string& yaml_file, const std::string& platform);
+        yaml_file_contents read_yaml_file(
+            const Context& ctx,
+            const std::string& yaml_file,
+            const std::string& platform,
+            bool use_uv
```

Comment:
Thank you for this contribution.

I think this change is not necessary since `use_uv` can be accessed from the `Context`.